versions/3.0.md

```diff
@@ -1464,7 +1464,7 @@ When passing complex objects in the `x-www-form-urlencoded` content type, the de
 
 ##### Special Considerations for `multipart` content
 
-It is common to use `multipart/form-data` as a `Content-Type` when transferring request bodies to operations.  In contrast to 2.0, a `schema` is required to define the input parameters to the operation when using `multipart` content.  This allows complex structures as well as supports mechanisms for multiple file uploads.
+It is common to use `multipart/form-data` as a `Content-Type` when transferring request bodies to operations.  In contrast to 2.0, a `schema` is required to define the input parameters to the operation when using `multipart` content.  This allows complex structures as well as supporting mechanisms for multiple file uploads.
```

Comment:
"What complex structures?"  The same complex stuff we use schemas in parameters/request body and response bodies.  e.g. JSON Objects, JSON Arrays of Objects.
"what supporting mechanisms" Encoding Object